content/influxdb/v2.0/get-started.md

```diff
@@ -28,9 +28,17 @@ _See [Differences between InfluxDB Cloud and InfluxDB OSS](#differences-between-
 
 <!-------------------------------- BEGIN macOS -------------------------------->
 {{% tab-content %}}
-### Download and install InfluxDB v2.0
+### Install InfluxDB v2.0
```

Comment:
I think we need to add some content sections under this heading:

- Use Homebrew
- Manually download and install

All of the existing instructions only apply to manually downloading and installing. Then added anchored links to each.